spec/acceptance/validate_metadata_spec.rb

```diff
@@ -70,4 +70,71 @@
       end
     end
   end
+
+  context 'when validating a specific file' do
+    include_context 'in a new module', 'foo'
```

Comment:
Due to SDK-294 (an intermittent bug when re-using the foo folder on Windows) it's preferable to give each feature its own test module name instead of always using 'foo'